### DISCUSSION TITLE
Github Education申請已經失敗20次，我還是找不到申請文件有哪裡有任何問題。

### DISCUSSION THREAD

**Discussion by sooophiahuang** [community/community#131251] [GitHub Education]
### Hi everyone,

您好，我的Github Education申請已經失敗20次了，
明明所有系統提示中要我更改或是修正的東西(Photo, Billing Information, Profile等) 都已經改完了，
但還是持續被拒絕。
照著大家在社群裡類似的提問試著解決過了，但仍舊沒有起效。
因此想請問是否有管道可以幫我進行人工審核，或是讓我更明確的知道整份申請文件哪裡有真正的問題呢？
謝謝您們。

**Comment by Er-Sadiq** [comment]
Hello,

If your Github Education application has been repeatedly rejected despite following all the guidelines, I recommend contacting Github Support directly for a manual review. Provide detailed information about your issue and any relevant documents. You can reach out to them through their support page: [GitHub Support](https://support.github.com/contact). This should help you get a clearer understanding of the problem and potentially resolve the issue.

Best regards.

**Comment by ossanna16** [comment]
Hi @sooophiahuang , thanks for participating!

👋 and thank you for being a part of our community. To ensure a safe and supportive environment, we look to our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) which states all communications here on the Community needs to be in English. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be closing and locking any posts in languages other than English for now, including this one.
